indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -180,6 +180,39 @@
   (or (not (empty? (:DirectDistributionInformation collection)))
       (tag/has-cloud-s3-tag? tags)))
 
+(def geoss-url-list
+  ["https://creativecommons.org/licenses/by/4.0/legalcode"
+   "http://creativecommons.org/licenses/by/4.0/"
+   "https://earthdata.nasa.gov/earth-observation-data/data-use-policy"
+   "https://science.nasa.gov/earth-science/earth-science-data/data-information-policy"
+   "https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1/sentinel-1-terms-conditions/"
+   "https://ladsweb.modaps.eosdis.nasa.gov/missions-and-measurements/sentinel3/terms-and-conditions/"])
+
+(defn- contains-geoss-url?
+  "Returns true if use-constraints fields contain urls in geoss-url-list."
+  [use-constraints]
+  (let [description (:Description use-constraints)
+        linkage (get-in use-constraints [:LicenseURL :Linkage])
+        license-text (:LicenseText use-constraints)]
+    (or (some #(str/includes? description %) geoss-url-list)
+        (some #(str/includes? linkage %) geoss-url-list)
+        (some #(str/includes? license-text %) geoss-url-list))))
+
+(defn- alter-consortiums
+  "Alter the consortiums list based on use-constraints."
```

Comment:
Add comment on why we add `GEOSS` to consortiums when there is a consortium of `EOSDIS` and no `use-constraints`.

Reply:
why don't we do the same when there is `use-constraints`, but no value for `FreeAndOpenData`?

Reply:
When there if consortium of EOSDIS, the provider is considered EOSDIS provider and they want to add GEOSS to the list.  I don't know the reason behind I just code according to the requirement Erich put in the ticket.

Reply:
Comment is not gonna help more than the code. EOSDIS provider requires GEOSS consortium when there is no constraint. That's exactly what the code says. I don't have any more info about it.